src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -214,9 +228,21 @@ void SetCommonContextAttributes(ContextBuilder builder)
                         builder.Add(ldServiceAccount.Build());
                     }
 
-                    if (_currentContext.OrganizationId.HasValue)
+                    if (overridingOrganizationId.HasValue || _currentContext.OrganizationId.HasValue)
```

Comment:
⚠️ Line 223 above is something I came upon and worries me because I can't recall why this was done. There isn't really a use case for service account contexts with targeting so far. Regardless, there's no service account ID in context, so should we use the user ID with a kind (that's the LD term for the type) of "service account"? I'd like to straighten this out.